frontend/src/components/elements/Buttons/ButtonLink/Button.styles.js

```diff
@@ -1,21 +1,22 @@
 import styled from 'styled-components';
 export const StyledButton = styled.button`
```

Comment:
Zmień button na Link z biblioteki styled-components lub React.
Dla Link w jsx zamiast onClick daj linkTo, gdzie będzie trzeba podać link (wewnętrzny lub zewnętrzny). Sprawdź jeszcze czy z # będzie to działać. Jeśli nie to poszukaj biblioteki Reacta z HashLink.
To samo zrób dla rezerwacji.